src/components/HomePage/Banner.js

```diff
@@ -210,7 +210,7 @@ function Banner() {
 				</NoSsr>
 
 				<Typography variant='h5' className={classes.text} component='h3'
-					style={{ fontWeight: 500, marginTop: 10 }}><b>Location:</b> The Hill, room TBD
+					style={{ fontWeight: 500, marginTop: 10 }}><b>Location:</b> Carnesale Commons, Palisades Room
```

Comment:
This is a nit but I think the banner would look a lot better if this text was right-aligned

Reply:
You're totally right